src/scenes/UserGalleryPage/UserGalleryCollections.tsx

```diff
@@ -82,7 +84,10 @@ function UserGalleryCollections({ galleryRef, queryRef, mobileLayout }: Props) {
       nonNullCollections.filter((collection) => {
         const isNotHidden = !collection.hidden;
         const hasTokens = collection.tokens?.length;
-        const hasWhitespace = collection.layout?.whitespace?.length;
+        const hasWhitespace =
+          collection.layout?.sectionLayout?.find((layout) => !!layout?.whitespace?.length) !==
+          undefined;
```

Comment:
nit: shouldn't need to compare to undefined

Reply:
removed